handMaxDepth should be changed to double

OpenARK/include/DetectionParams.h:110:28: warning: implicit conversion from 'double' to 'int' changes value from 1.6 to 1 [-Wliteral-conversion]
        int handMaxDepth = 1.6;

--- a/include/DetectionParams.h
+++ b/include/DetectionParams.h
@@ -107,7 +107,7 @@ namespace ark {
         /**
          * maximum distance of hand from the camera.
          */
-        int handMaxDepth = 1.6;
+        double handMaxDepth = 1.6;
 
         /**
          * Amount to erode the contour image to remove small points

